Backpack/src/main/res/values/attrs.xml

```diff
@@ -293,6 +293,34 @@
     <attr name="state_error" format="boolean" />
   </declare-styleable>
 
+  <declare-styleable name="BpkImageSkeleton">
+    <attr name="cornerType" format="enum">
```

Comment:
might be worth prefixing that with `skeleton` as well to avoid running into conflicts with future components

Reply:
the prefixes here would still be good to address to avoid issues with future components (basically rename to `skeletonCornerType`. same for `diameter` and `circleSize below)